arch/arm/src/stm32f7/stm32_adc.c

```diff
@@ -735,7 +735,7 @@ static int adc_timinit(FAR struct stm32_dev_s *priv)
    *   position.
    */
 
-  ainfo("Initializing timers extsel = 0x%08x\n", priv->extsel);
+  ainfo("Initializing timers extsel = 0x%08lx\n", priv->extsel);
```

Comment:
```suggestion
  ainfo("Initializing timers extsel = 0x%08" PRIx32 " \n", priv->extsel);
```
Since you are fixing this, the most correct fix is to use the format specfiers from `inttypes.h`.

Reply:
@gustavonihei  I asked a question on the usage in the low level code [here]( https://github.com/apache/incubator-nuttx/pull/3169#issuecomment-806025894)

Would mind responding there as well. I am just looking for clarity on establishing the pattern and applying it consistently. 

Is it as simple as all format specifiers should be replaced everywhere?

Reply:
@davids5 I posted my answer there: https://github.com/apache/incubator-nuttx/pull/3169#issuecomment-806616431

So, in summary, for achieving the most standards-compliant and portable code:
- Native integer types (int, unsigned...) -> Specifiers from `fprintf` documentation (d, u....)
- Fixed width integer types from `stdint.h` (int32_t, uint32_t...): Specifiers from `inttypes.h` (PRId32, PRIu32)

Reply:
Thank you @gustavonihei!